code/john/gmbfnt/gmbfnt.c

```diff
@@ -68,15 +72,15 @@ void _splitpath(char *in,char *drive,char *dir,char *name,char *ext)
 	memcpy(dir,in,dirsize);
 	dir[dirsize]=0;
 }
-void strupr(char *s)
+char* strupr(char *s)
 {
 	while(*s)
 	{
 		*s=toupper(*s);
 		++s;
 	}
 }
-int strcmpi(char *s1,char *s2)
+int strcmpi(const char *s1,const char *s2)
```

Comment:
don't touch this

Reply:
All these changes in the string manipulation functions were done to match msys2 and mingw64 definitions in 'strings.h', otherwise it won't compile in Windows giving `error: conflicting types`, or, if I don't include 'string.h', segfaulting when used.